app/assets/javascripts/discourse/app/controllers/bookmark.js

```diff
@@ -89,6 +90,12 @@ export default Controller.extend(ModalFunctionality, {
     if (this._editingExistingBookmark()) {
       this._initializeExistingBookmarkData();
     }
+
+    scheduleOnce("afterRender", () => {
+      if (this.site.isMobileDevice) {
+        $("#bookmark-name").blur();
```

Comment:
I would prefer if you would try to use vanilla js here, this should be simple enough to do